snow/validators/state.go

```diff
@@ -24,6 +24,28 @@ type State interface {
 	// GetSubnetID returns the subnetID of the provided chain.
 	GetSubnetID(ctx context.Context, chainID ids.ID) (ids.ID, error)
 
+	// GetWarpValidatorSets returns the canonical warp validator set for all
+	// subnets at the requested P-chain height.
+	//
+	// If a subnet is not present in the returned map, that indicates that the
+	// subnet is not currently able to send warp messages.
```

Comment:
```suggestion
	// subnet is not currently able to produce valid warp message signatures.
```

Nit: "send" is a bit of misnomer IMO